.changeset/green-carrots-scream.md

```diff
@@ -0,0 +1,12 @@
+---
+"wrangler": patch
+---
+
+feat: Add support for "json" bindings
+
+Did you know? We have support for "json" bindings! Here are a few examples:
+
+[vars]
+text = "plain ol' string"
+count = 1
+complex = { enabled = true, id = 123 }
```

Comment:
Strictly this object literal is not JSON (since it is in TOML).
Is the fact that it will be stringified to JSON in order to upload it is an implementation concern?
Perhaps we should just say that we now support more complex literals? Or perhaps more nuanced bindings that can be stringified as JSON?